test/distributed/fsdp/test_fsdp_param_exec_order_wrap.py

```diff
@@ -58,26 +67,59 @@ def device(self):
         [ShardingStrategy.FULL_SHARD, ShardingStrategy.SHARD_GRAD_OP],
     )
     @parametrize("iters", [1, 3])
-    def test_fsdp_flatten_params_exec_order(self, sharding_strategy: ShardingStrategy, iters: int):
+    def test_fsdp_flatten_params_exec_order(
+        self,
+        sharding_strategy: ShardingStrategy,
+        iters: int,
```

Comment:
I think we mentioned this in the last PR, but eventually, we do not test need to test both `iters=1` and `iters=3` right?

Reply:
makes sense, here I removed the `iters` arg, and just run one forward and backward pass in the test without a for loop